package.json

```diff
@@ -45,11 +46,11 @@
     "eslint-config-bliss": "^5.0.0",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-react": "^7.23.2",
-    "husky": "5",
+    "husky": "^7",
     "lint-staged": "^10.5.4",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "react-syntax-highlighter": "^13.5.1",
+    "react": "^17.0.2",
```

Comment:
Upgrading these makes them match the version in `devDependencies`, which fixes some Yarn warnings that were polluting the command-line output.